tests/curl/curl.go

```diff
@@ -0,0 +1,67 @@
+package curl
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/kudobuilder/test-tools/pkg/client"
+	"github.com/kudobuilder/test-tools/pkg/cmd"
+	"github.com/kudobuilder/test-tools/pkg/kubernetes"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/uuid"
+)
+
+func RunCommand(client client.Client, namespace string, arguments ...string) (string, string, error) {
+	curlPodName := fmt.Sprintf("curl-%s", uuid.NewUUID())
+
+	podTpl := v1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      curlPodName,
+			Namespace: namespace,
+		},
+		Spec: v1.PodSpec{
+			Containers: []v1.Container{
+				{
+					Name:    "curl",
+					Image:   "curlimages/curl",
+					Command: []string{"sleep", "300"},
+				},
+			},
+		},
+	}
+
+	pod, err := kubernetes.NewPod(client, podTpl)
+	if err != nil {
+		return "", "", fmt.Errorf("failed to create new curl pod: %v", err)
+	}
+	defer func() {
+		_ = pod.Delete()
```

Comment:
Nit: Let's at least log something if the pod deletion fails. Or use named result value as a workaround to capture the error like in this example: https://play.golang.org/p/PZGn5-1TXv4.

Reply:
Fancy. Didn't know you could do that. I added a log though, as it's a warning at most and nothing critical.